web/src/app/schedules/calendar/ScheduleCalendar.tsx

```diff
@@ -364,7 +364,7 @@ function ScheduleCalendar(props: ScheduleCalendarProps): JSX.Element {
               startIcon={<GroupAdd />}
               title='Make temporary change to schedule'
             >
-              Override
+              Add Override
```

Comment:
To completely avoid the overlap with ADD, how do you feel about having this just be `Override`? as in "override what you see on the calendar"